src/components/AddOn/core/AddOnEntry/AddOnEntry.test.tsx

```diff
@@ -30,9 +30,9 @@ describe('Testing AddOnEntry', () => {
   const props = {
     id: 'string',
     enabled: true,
-    title: 'string',
-    description: 'string',
-    createdBy: 'string',
+    title: 'title',
```

Comment:
the `title, description, and createdBy` field shouldn't be changed. This object is only showing what types each field should have.  You can still create your test without changing these fields